core/src/components/split-pane/split-pane.md.scss

```
@@ -19,6 +24,11 @@
 }
 
 :host(.split-pane-visible) ::slotted(.split-pane-side[side=end]) {
+  @include rtl() {
+    border-right: var(--border);
+    border-left: 0;
+  }
+  
```

```suggestion
  @include border(0, 0, 0, var(--border));
```

Let's remove the `@include rtl()` stuff and replace it with the mixin.

You will also need to get rid of the `border-left: var(--border);` and `border-right: 0` lines below.